examples/config.js

```diff
@@ -17,6 +17,19 @@ module.exports = {
 
     // Enable caching of connections for all scenarios in trRouting. Will use more memory
     trRoutingCacheAllScenarios: false,
+    // Extra options for running trRouting
```

Comment:
We if add a config block for trRouting, then we need to move the cacheAllFlag inside of it.

Reply:
We should also move the port number there.

Reply:
Done in previous PR